lib/companies/common.ts

```diff
@@ -0,0 +1,4 @@
+import { DateTime } from "luxon";
+import { TIME_ZONE } from "../../config/config";
+
+export const mondayOffset = DateTime.now().setZone(TIME_ZONE).weekday - 1;
```

Comment:
Might want this to be a function?

Reply:
Why is that? It should compute upon page load anyways, so should be correct every time afaik.

Reply:
Considering that the value is passed to `fetchScheduleWithDayOffset`, which computes `DateTime.now()` for `startDate` (i.e. could potentially used different day for "now"), I think it would be clearer for it to be computed when used.

Reply:
Sure. Changed it now 😄